src/delete.js

```diff
@@ -0,0 +1,36 @@
+export default class Delete {
+  // delete list
+  trash(col) {
+    const parent = document.querySelector('.output');
+    parent.addEventListener('click', (e) => {
+      if (e.target.classList.contains('fa-trash')) {
+        this.clearStorage(col, e.target.id);
+        e.target.parentElement.remove();
+      }
+    });
+  }
+
+  clearStorage(col, id) {
+    this.col = col;
+    this.col.forEach((el, index) => {
+      if (el.index === Number(id)) {
+        this.col.splice(index, 1);
+      }
+    });
+    localStorage.setItem('list', JSON.stringify(this.col));
+  }
```

Comment:
- [ ] Deleting completed tasks should update all remaining items' indexes, so they represent the current list order and are unique.